bin/.rustup-init-1.25.1.pkg

```diff
@@ -0,0 +1 @@
+hermit
```

Comment:
i think the key thing i was missing was the `rust-toolchain.toml`- i had dep problems with `rustup` alone because it wasn't initializing the toolchain